### PR TITLE
Update all npm package

### PR DESCRIPTION
**Pull request by shreyasikhar**
This PR adds following changes:
1. Updates all npm packages to their latest version
2. Re-runs npm build commands
3. Adds `.nvmrc` file to store required node version

### CODE REVIEW

File: assets/.nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (gagan0123) [inline assets/.nvmrc:1]: Do we need Node 12 or will things work with Node 14 as well?

File: languages/wp-menu-custom-fields.po [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2020 rtCamp
+# Copyright (C) 2022 rtCamp
```
Comment (gagan0123) [inline languages/wp-menu-custom-fields.po:1]: This file can be skipped, since there are no changes to strings.

**Review by gagan0123 — CHANGES_REQUESTED** [review]

File: assets/.nvmrc (hunk 1 above)
Comment (shreyasikhar) [reply assets/.nvmrc]: Updated to 14

File: languages/wp-menu-custom-fields.po (hunk 2 above)
Comment (shreyasikhar) [reply languages/wp-menu-custom-fields.po]: Removed this file now, thanks 👍

**Review by gagan0123 — APPROVED** [review]

**Pull request merged**